components/MemberDirectory.tsx

```diff
@@ -34,14 +48,36 @@ export default function MemberDirectory({ members }: MemberDirectoryProps) {
               <h2 className="member__name">{member.name}</h2>
               <div className="member__location">
                 <h3>{member.location}</h3>
-                <h4>{member.region}</h4>
+                {member.regionFilter[0] ? (
+                  <h4>
+                    <Pill
+                      active={member.regionFilter[0].active}
+                      customWidth="max-content"
+                      customFontSize="1rem"
```

Comment:
Curious whatcha thinking with this new `customFontSize` prop?

<img width="694" alt="Screen Shot 2022-10-12 at 12 15 59 AM" src="https://user-images.githubusercontent.com/5141111/195275955-d7245207-335f-46e4-9253-b7ef32cc5162.png">

I worry it creates an inconsistent size between the two pill groups. It looks a bit awkward as they're so close to each other. Especially when their closest adjacent sibling (the location or job title) are otherwise the same.

Reply:
@taylorkmho I had a few things in mind with this one: 

1. I was trying to keep it a bit consistent with the sizing of the original Region text, it is slightly bigger than the filter pills on the current implementation

<img width="967" alt="Screen Shot 2022-10-12 at 8 23 51 PM" src="https://user-images.githubusercontent.com/26368729/195491862-c69f1e92-d9cb-4b02-b1c5-ea94a4a8d55f.png">

2. It feels a _tiny_ bit small when it's at the original size in comparison to the **location** right above it. In my eyes, some of the location names aren't as familiar (e.g. Bend or Lehi) with me so it feels like the **region** is more of an aside from the **location** which is visually hard to see when the **region** is much smaller, yet the **region** is the filter that groups most of the locations up. 

<img width="855" alt="Screen Shot 2022-10-12 at 8 26 47 PM" src="https://user-images.githubusercontent.com/26368729/195492243-bca564e8-1b88-4856-bd47-06b7ebff6024.png">

I don't have strong feelings about the sizing though. Maybe changing the sizing or ordering of the location might be better but either way, I'm definitely down to change it so I'll leave the decision up to you.

Reply:
Any thoughts on not using a tag for the region? I understand we want to highlight selected criteria or make it obvious, which the Pill pattern comes with (with its "Selected" state). Either way, but I do agree that it does feel a bit "backwards" when it's smaller. I personally read into it as more being due to it being a tag/pill than the size.